commit a921fdc1a5d565aa7c3258487b90ee86d26dc510
Author: Greg Kroah-Hartman <gregkh@suse.de>
Date:   Tue Oct 5 09:41:45 2010 -0700

    Staging: brcm80211: remove floating point typedefs
    
    It's not ever used, so remove the typedef.  Floating point isn't used
    in the kernel, so this could never be an issue here.
    
    Cc: Brett Rudley <brudley@broadcom.com>
    Cc: Henry Ptasinski <henryp@broadcom.com>
    Cc: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/include/typedefs.h
+++ b/drivers/staging/brcm80211/include/typedefs.h
@@ -109,31 +109,6 @@ typedef signed int int32;
 typedef signed long long int64;
 #endif
 
-/* define float32/64, float_t */
-
-#ifndef TYPEDEF_FLOAT32
-typedef float float32;
-#endif
-
-#ifndef TYPEDEF_FLOAT64
-typedef double float64;
-#endif
-
-/*
- * abstracted floating point type allows for compile time selection of
- * single or double precision arithmetic.  Compiling with -DFLOAT32
- * selects single precision; the default is double precision.
- */
-
-#ifndef TYPEDEF_FLOAT_T
-
-#if defined(FLOAT32)
-typedef float32 float_t;
-#else				/* default to double precision floating point */
-typedef float64 float_t;
-#endif
-
-#endif				/* TYPEDEF_FLOAT_T */
 
 /* define macro values */
 
